content/applications/general/email_communication/email_domain.rst

```diff
@@ -1,163 +1,190 @@
-========================
-Sending emails with Odoo
-========================
+=======================
+Send an email with Odoo
+=======================
 
-Using your email domain in Odoo
-===============================
+Use an email domain in Odoo
+===========================
 
 Documents in Odoo (a CRM opportunity, a sales order, an invoice ...) have a discussion thread,
 called *chatter*.
 
-When you post a message in the chatter, this message is sent by email to the followers of the
-document. If a follower replies to the message, the reply updates the chatter, and Odoo relays the
-reply to the followers.
+When a database user posts a message in the chatter, this message is sent by email to the followers
+of the document. If a follower replies to the message, the reply updates the chatter, and Odoo
+relays the reply to the followers.
 
-Emails from your users to partners (customers, vendors) are sent from the email address of your
-users. Similarly, emails from partners to users are sent from the email address of the partners.
-This allows you to recognize at a glance who sent an email relayed by Odoo.
+Emails from the database users to partners (customers or vendors) are sent from the email addresses
+of the database users. Similarly, emails from partners to users are sent from the email addresses
+of the partners. That way, the database users can recognize who sent an email relayed by Odoo.
```

Comment:
Suggestion to cut down on filler and improve readability. 

```suggestion
Messages sent in the chatter from internal database users to external users (such as partners,
customers, or vendors) are sent from the email addresses of the database users. Similarly, messages
sent back to the chatter from external users will appear from their respective email addresses, or
as they are listed in their Contacts record.
```

Reply:
I made some edits to this. 

> Messages sent in the chatter from internal database users to external users (such as partners,
customers, or vendors) are relayed on behalf of the database users. Messages
sent back to the chatter from external users will appear from their respective email addresses, or
as they are listed in their Contacts record.

>